tests/admin_docs/tests.py

```diff
@@ -390,6 +390,8 @@ def test_model_docstring_renders_correctly(self):
             'reference external" href="/admindocs/models/myapp.company/">'
             'myapp.Company</a> where the person works.</dd></dl>'
         )
+        descriptionheading = '<h3>Description</h3>'
```

Comment:
No, but maybe it'd be better placed there indeed as this stuff isn't actually about models (or descriptions for that matter). Looks like moving it to where you suggest would require changes in several TestUtils.test_*() methods though.